packages/blade/src/utils/mapProps/accessibilityMap.web.ts

```diff
@@ -0,0 +1,68 @@
+import type { CommonAccessibilityKeys, CommonAccessibilityProps } from './types';
+
+export const accessibilityValue = {
+  accessibilityValueMax: 'aria-valuemax',
+  accessibilityValueMin: 'aria-valuemin',
+  accessibilityValueNow: 'aria-valuenow',
+  accessibilityValueText: 'aria-valuetext',
+};
+
+export const accessibilityState = {
+  accessibilitySelected: 'aria-selected',
+  accessibilityDisabled: 'aria-disabled',
+  accessibilityExpanded: 'aria-expanded',
+  accessibilityBusy: 'aria-busy',
+  accessibilityChecked: 'aria-checked',
+};
+
+// TODO:
+// importantForAccessibility
+// accessibilityElementsHidden
+// accessibilityViewIsModal
+export const accessibilityMap: AccessibilityMap = {
+  ...accessibilityState,
+  ...accessibilityValue,
+  accessibilityActiveDescendant: 'aria-activedescendant',
+  accessibilityAtomic: 'aria-atomic',
+  accessibilityAutoComplete: 'aria-autocomplete',
+  accessibilityColCount: 'aria-colcount',
+  accessibilityColIndex: 'aria-colindex',
+  accessibilityColSpan: 'aria-colspan',
+  accessibilityControls: 'aria-controls',
+  accessibilityDescribedBy: 'aria-describedby',
+  accessibilityDetails: 'aria-details',
+  accessibilityErrorMessage: 'aria-errormessage',
+  accessibilityFlowTo: 'aria-flowto',
+  accessibilityHasPopup: 'aria-haspopup',
+  accessibilityHidden: 'aria-hidden',
+  accessibilityInvalid: 'aria-invalid',
+  accessibilityKeyShortcuts: 'aria-keyshortcuts',
+  accessibilityLabel: 'aria-label',
+  accessibilityLabelledBy: 'aria-labelledby',
+  accessibilityLiveRegion: 'aria-live',
+  accessibilityModal: 'aria-modal',
+  accessibilityMultiline: 'aria-multiline',
+  accessibilityMultiSelectable: 'aria-multiselectable',
+  accessibilityOrientation: 'aria-orientation',
+  accessibilityOwns: 'aria-owns',
+  accessibilityPlaceholder: 'aria-placeholder',
+  accessibilityPosInSet: 'aria-posinset',
+  accessibilityPressed: 'aria-pressed',
+  accessibilityReadOnly: 'aria-readonly',
+  accessibilityRequired: 'aria-required',
+  accessibilityRole: 'role',
+  accessibilityRoleDescription: 'aria-roledescription',
+  accessibilityRowCount: 'aria-rowcount',
+  accessibilityRowIndex: 'aria-rowindex',
+  accessibilityRowSpan: 'aria-rowspan',
+  accessibilitySetSize: 'aria-setsize',
+  accessibilitySort: 'aria-sort',
+  accessibilityCurrent: 'aria-current',
+  accessibilityDropEffect: 'aria-dropeffect',
+  accessibilityGrabbed: 'aria-grabbed',
+  accessibilityLevel: 'aria-level',
+  accessibilityRelevant: 'aria-relevant',
```

Comment:
Why are we appending `accessibility` to all the keys? Lets just remove these keys? Much simpler API. The function name `mapA11yProps` or whatever we decide to call it should be a good enough context on what these keys are

Reply:
Not sure how expressive that would be, With `accessibility` prefixes the API is same as [`react-native-web`](https://necolas.github.io/react-native-web/docs/accessibility/), [`react-native`](https://reactnative.dev/docs/accessibility) & web also prefixes with `aria-` 

Just that working with `aria` and accessibility it just feel kinda unusual to not have the prefix of some kind. 
```ts
  return mapAccessibilityProps({
    role: 'checkbox',
    disabled: isDisabled,
    checked: isChecked,
  });
```

Reply:
Decided to go with makeAccessible and no accessibility prefix.